config/config.go

```diff
@@ -42,3 +43,8 @@ func LoadConfig(file string) {
 		log.SetLevel(log.ErrorLevel)
 	}
 }
+
+// AbsolutePath returns the absolute file path under the specified data directory
+func AbsolutePath(name string) string {
```

Comment:
I would suggest use `DataDir(name string)`, `AbsoluteDataFile()` or any other function names. The sense is that the function name `AbsolutePath` is widely used in other libraries and most of them is to convert relative path into absolutely path in OS manner.